src/popup/components/InputField.vue

```diff
@@ -167,7 +168,12 @@ export default defineComponent({
     const inputEl = ref<HTMLInputElement | null>(null);
     const isBlinking = ref(false);
 
-    const inputMode = computed(() => props.type === 'number' ? 'decimal' : 'text');
+    const inputMode = computed(() => ({
+      number: 'decimal',
+      url: 'url',
+      text: 'text',
+    }[props.type]));
+    const autoCapitalize = computed(() => props.type === 'url' ? 'off' : undefined);
```

Comment:
I don't fully understand what is the goal for this line. Maybe you can add comment here?

Reply:
The idea is to set `autocapitalize="off"` in case `inputmode="url"`, to don't begin with a capital letter on iOS

Reply:
Sure. Makes sense. Can you please add a comment about this?